src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -33,7 +33,7 @@ const Body = BaseFormWithPolling.extend(
     initialize() {
       BaseFormWithPolling.prototype.initialize.apply(this, arguments);
       this.listenTo(this.model, 'error', this.onPollingFail);
-      doChallenge(this);
+      doChallenge(this, this.options.settings.identifier);
```

Comment:
you can remove the extra argument.. because we are passing `this` and it already contain the value you need. it can be used in `doChallenge`

`const hint = this.option.settings?.identifier || ''`